### PR TITLE
feat: make electron windows build

### PR DESCRIPTION
**Pull request by m1911star**
- make windows build work
- add windows release workflow
- fix MacOS intel release version not working

### CODE REVIEW

File: apps/electron/scripts/common.mjs [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import fs from 'node:fs';
 import path from 'node:path';
-
-const __dirname = new URL('.', import.meta.url).pathname;
```
Comment (m1911star) [inline apps/electron/scripts/common.mjs:4]: this will not work on windows

File: apps/electron/package.json [hunk 2]
```diff
@@ -3,12 +3,17 @@
   "productName": "AFFiNE",
   "private": true,
   "version": "0.0.0",
+  "author": "affine",
+  "description": "AFFiNE App",
+  "homepage": "https://github.com/toeverything/AFFiNE",
   "scripts": {
-    "dev": "node scripts/dev.mjs",
-    "prod": "NODE_ENV=production node scripts/dev.mjs",
+    "dev": "cross-env NODE_ENV=development node scripts/dev.mjs",
```
Comment (m1911star) [inline apps/electron/package.json:10]: add cross env to make windows right

File: apps/electron/forge.config.js [hunk 3]
```diff
@@ -12,5 +12,13 @@ module.exports = {
         name: 'AFFiNE',
       },
     },
+    {
+      name: '@electron-forge/maker-zip',
```
Comment (m1911star) [inline apps/electron/forge.config.js:16]: release windows zip version for now

File: .github/workflows/client-app.yml [hunk 4]
```diff
@@ -91,7 +91,7 @@ jobs:
           path: apps/electron/dist
 
       - name: make build
-        run: yarn make-macos-arm64
+        run: yarn make-macos-x64
```
Comment (pengx17) [inline .github/workflows/client-app.yml:94]: ops

**Review by pengx17 — APPROVED** [review]
LGTM

**Pull request merged**